assets/js/blocks/product-query/constants.ts

```diff
@@ -90,7 +90,15 @@ export const INNER_BLOCKS_TEMPLATE: InnerBlockTemplate[] = [
 			],
 			[
 				'woocommerce/product-price',
-				{ textAlign: 'center', fontSize: 'small' },
+				{
+					textAlign: 'center',
+					fontSize: 'small',
+					style: {
+						spacing: {
+							margin: { bottom: 'var:preset|spacing|30' },
```

Comment:
Just so I learn more about this API, how does this work?

Reply:
It pre-sets the bottom margin to `1`:
![Edit_Page_“Product_Query”_‹_ratings_—_WordPress](https://user-images.githubusercontent.com/905781/213203186-6ce28be6-71db-4075-9264-1e317b58c13d.jpg)